packages/desktop-client/src/components/modals/CreateLocalAccountModal.tsx

```diff
@@ -49,7 +55,12 @@ export function CreateLocalAccountModal({
               event.preventDefault();
 
               const nameError = !name;
-              setNameError(nameError);
+              setNameError({ error: nameError, message: 'Name is required' });
+
+              if (accounts.some(account => account.name === name)) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I wonder if we should make this case insensitive. WDYT?

Reply:
I'd leave that up to the user. I'd say more control is better than less control, but that is just my opinion. LMK if we should change. Can always be done later.